frontend/src/metabase/dashboard/containers/DashboardHeader.jsx

```diff
@@ -344,19 +339,17 @@ class DashboardHeader extends Component {
             onDeleteBookmark={deleteBookmark}
             isBookmarked={isBookmarked}
           />,
-          <DashboardHeaderInfoButton
-            key="dashboard-info-button"
-            icon="info"
-            iconSize={18}
-            onlyIcon
-            borderless
-            isShowingDashboardInfoSidebar={isShowingDashboardInfoSidebar}
-            onClick={() =>
-              isShowingDashboardInfoSidebar
-                ? closeSidebar()
-                : setSidebar({ name: SIDEBAR_NAME.info })
-            }
-          />,
+          <Tooltip key="dashboard-info-button" tooltip={t`More info`}>
```

Comment:
Add a missing tooltip to match the button on questions.
#### Dashboard
![image](https://user-images.githubusercontent.com/1937582/178477618-7cfa6fc5-ab72-4609-9ca3-abae47232643.png)

#### Questions
![image](https://user-images.githubusercontent.com/1937582/178477663-8c065968-2434-495d-b29c-0a30d7aa24da.png)